Rohit/PostgresAssignment/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "api_myschool",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "@koa/router": "^10.0.0",
+    "dotenv": "^10.0.0",
+    "joi": "^17.4.1",
+    "koa": "^2.13.1",
+    "koa-bodyparser": "^4.3.0",
+    "koa-json": "^2.0.2",
+    "nodemon": "^2.0.12",
+    "pg": "^8.6.0"
+  },
+  "devDependencies": {
+    "@types/node": "^15.14.0",
+    "@types/pg": "^8.6.1",
+    "@typescript-eslint/eslint-plugin": "^4.28.3",
+    "@typescript-eslint/parser": "^4.28.3",
+    "eslint": "^7.30.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.23.4"
+  },
+  "scripts": {
+    "dev": "nodemon src/app/server.ts",
```

Comment:
you can also add other scripts - ref : https://github.com/adityajawade93/freshers-june2021/tree/main/jatin/postgres_assignment

Reply:
added more scripts